include/openthread/thread.h

```diff
@@ -505,6 +505,48 @@ const char *otThreadGetDomainName(otInstance *aInstance);
  */
 otError otThreadSetDomainName(otInstance *aInstance, const char *aDomainName);
 
+/**
+ * Set the Thread Domain Unicast Address manually.
+ *
+ * This function is only available since Thread 1.2 when `OPENTHREAD_CONFIG_DUA_ENABLE` is enabled.
+ *
+ * @param[in]  aInstance             A pointer to an OpenThread instance.
+ * @param[in]  aDomainUnicastAddress A pointer to the Thread Domain Unicast Address to set.
+ *
+ * @sa otThreadGetFixedDomainUnicastAddress
+ * @sa otThreadClearFixedDomainUnicastAddress
+ */
+void otThreadSetFixedDomainUnicastAddress(otInstance *aInstance, const otIp6Address *aDomainUnicastAddress);
+
+/**
+ * Get the Thread Domain Unicast Address which was set manually.
+ *
+ * This function is only available since Thread 1.2 when `OPENTHREAD_CONFIG_DUA_ENABLE` is enabled.
+ *
+ * @param[in]  aInstance A pointer to an OpenThread instance.
+ *
+ * @returns A pointer to the Thread Domain Unicast Address which was set manually, or
+ *          NULL if none was set manually.
+ *
+ * @sa otThreadSetFixedDomainUnicastAddress
+ * @sa otThreadClearFixedDomainUnicastAddress
+ *
+ */
+const otIp6Address *otThreadGetFixedDomainUnicastAddress(otInstance *aInstance);
+
+/**
+ * Clear the Thread Domain Unicast Address which was set manually if any.
+ *
+ * This function is only available since Thread 1.2 when `OPENTHREAD_CONFIG_DUA_ENABLE` is enabled.
+ *
+ * @param[in]  aInstance A pointer to an OpenThread instance.
+ *
+ * @sa otThreadSetFixedDomainUnicastAddress
+ * @sa otThreadGetFixedDomainUnicastAddress
+ *
+ */
+void otThreadClearFixedDomainUnicastAddress(otInstance *aInstance);
```

Comment:
We can just use `otThreadSetFixedDomainUnicastAddress(instance, NULL)` for this purpose ?

Reply:
Considered possible extension of directly setting iid in the future as explained below comment https://github.com/openthread/openthread/pull/4900/commits/8524be9b3c669d3ffd2f7f03bf73f72bad7a66b9#r417084715, maybe keep this API but rename it as `otThreadClearFixedDomainUnicastAddressConfig()`?